lib/client.ts

```diff
@@ -133,7 +133,10 @@ class ClientManager extends EventTarget {
 
   private async _getConfig(ozoneDid?: string) {
     const builtIn =
-      ozoneDid || process.env.NEXT_PUBLIC_OZONE_SERVICE_DID || undefined
+      ozoneDid ||
+      process.env.NEXT_PUBLIC_OZONE_SERVICE_DID ||
+      'did:plc:kqg2icildynagykzlxehkkaw' ||
```

Comment:
I think this may be left around from dev.

Reply:
yep. cleaned up!